ledger/src/blockstore.rs

```diff
@@ -2755,12 +2755,7 @@ impl Blockstore {
             .zip(slot_metas)
```

Comment:
it looks like `get_slots_since` isn't used for rpc. So, it's only used by replay stage.